tests/test_tacotron2_model.py

```diff
@@ -104,7 +104,7 @@ def test_train_step():
             count += 1
```

Comment:
It is interesting to add a unit test like MultiSpeakeTacotronTrainTest to the Multilingual model as well so when it breaks we will know

Reply:
Yeah I know I planed to add tests before removing the draft status of the PR